runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -44,13 +51,16 @@ TR_VectorAPIExpansion::perform()
    {
    bool disableVectorAPIExpansion = comp()->getOption(TR_DisableVectorAPIExpansion);
    bool traceVectorAPIExpansion = comp()->getOption(TR_TraceVectorAPIExpansion);
+   _boxingAllowed = comp()->getOption(TR_EnableVectorAPIBoxing);
 
    _trace = traceVectorAPIExpansion;
 
    if (J2SE_VERSION(TR::Compiler->javaVM) >= J2SE_V17 &&
        !disableVectorAPIExpansion &&
        !TR::Compiler->om.canGenerateArraylets() &&
-       findVectorMethods(comp()))
+       findVectorMethods(comp())
+       && !comp()->isDLT() // TODO
```

Comment:
Please place the `&&` at the end of the previous line, in the same way as other lines in this expression.

Reply:
done